AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/QtPyScriptCanvasEditor.py

```diff
@@ -0,0 +1,48 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Object to house all the Qt Objects used when testing and manipulating the O3DE UI
+"""
+from PySide2 import QtWidgets
+from consts.scripting import (SCRIPT_CANVAS_UI)
+
+
+class QtPyScriptCanvasEditor:
+    """
+    QtPy class for handling the behavior of the Script Canvas editor.
+
+    Note: This class contains a reference to the SC editor's main window AND the SC editor itself. This was done because the other
+    QtPy elements such as the SC variable manager can be toggled on and off by the user but the main pane cannot. If a need
+    to separate the two objects is discovered the main pane object will be put into its own QtPy class
+    """
+
+    def __init__(self, editor_main_window):
+        self.sc_editor = editor_main_window.findChild(QtWidgets.QDockWidget, SCRIPT_CANVAS_UI)
+        self.sc_editor_main_window = self.sc_editor.findChild(QtWidgets.QMainWindow)
+
+    def get_sc_main_pane(self):
```

Comment:
Let's rename this to be more meaningful.

Since we're now in a QTPyScriptCanvasEditor class that provides context for readability, we can rename it to something like, `get_main_pane/get_main_panel`